riemann/jar/src/sixsq/slipstream/api/service_offer.clj

```diff
@@ -0,0 +1,51 @@
+(ns sixsq.slipstream.api.service-offer
+    (:require
+    [environ.core :as env]
+    [clj-http.client :as client]
+    [clojure.data.json :as json]))
+
+(defn- json->edn [json]
+  (when json (json/read-str json :key-fn keyword)))
+
+(defn edn->json [edn]
+  (json/write-str edn))
+
+(defn- name-id
+  [connector]
+  [(get-in connector [:connector :href]) (:id connector)])
+
+(defn- record-connector
+  [m [name id]]
+  (assoc m (keyword name) {:id id :state :nok}))
+
+(def slipstream-endpoint        (env/env :slipstream-endpoint))
+(def slipstream-super-password  (env/env :slipstream-super-password))
+
+(println "Using SlipStream endpoint : " slipstream-endpoint)
+
+(defn- cookie-store-authn
+  []
+  (let [my-cs (clj-http.cookies/cookie-store)
+        _ (client/post (str slipstream-endpoint "/auth/login?user-name=super&password=" slipstream-super-password)
```

Comment:
Yes, future improvement would be to have a dedicated user with `admin-service-offer` role.